module/module_calls.go

```diff
@@ -42,3 +42,14 @@ func (lsa LocalSourceAddr) ForDisplay() string {
 func (lsa LocalSourceAddr) String() string {
 	return string(lsa)
 }
+
+type UnknownSourceAddr struct {
```

Comment:
I know I originally proposed this struct 😅 but now that I see it I wonder if it's better to just make it `type UnknownSourceAddr string`, similar to `LocalSourceAddr`?